client/src/pages/BoardsPage/BoardsPage.js

```diff
@@ -120,19 +114,83 @@ const BoardsPage = _ => {
     }
   }
 
-  boardState.boardSync = () => {
-    const boardLength = JSON.parse(localStorage.getItem('boards'))
-
-    if (boardState.boards.length === boardLength) {
-      return false
+  boardState.handleSubmitBoard = e => {
```

Comment:
This function came from the BottomBarPage. Putting it here because we moved the button from the bottom bar to the board page.